lms/models/application_instance.py

```diff
@@ -179,52 +174,3 @@ def product(self):
             product = self.Product.UNKNOWN
 
         return product
-
-    @classmethod
-    def build_from_lms_url(  # pylint:disable=too-many-arguments
```

Comment:
Moved to ApplicationInstanceService.

Reply:
Yep, this is a good fix. This doesn't look like model code.